src/components/CallView/Grid/Grid.vue

```diff
@@ -112,6 +112,7 @@
 							class="video"
 							:is-grid="true"
 							:fit-video="isStripe"
+							:token="token"
```

Comment:
Can't you read it from the store instead of passing it through X layers?

Reply:
It is a matter of taste, I guess :shrug: I prefer to pass data from the parent views rather than using the store. Anyway I can change it.